### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by luocmin** [zylo117/Yet-Another-EfficientDet-Pytorch#511]
I trained myself in 18,300 pieces of training sets, and the classification loss was especially great at the beginning of 47932.41406, and the regression loss was 1.6. After the training of an epoch, the classification loss was 15. Is this normal

**Comment by zylo117** [comment]
Yes

**Comment by LoveIsAGame** [comment]
During my training,my reg_loss remain unchanged，Do you know the possible reason？
